rwgame/main.cpp

```diff
@@ -1,11 +1,10 @@
 #include <iostream>
 #include "RWGame.hpp"
 #include "SDL.h"
-
 #include <core/Logger.hpp>
 
 int main(int argc, char* argv[]) {
-    // Initialise Logging before anything else happens
+    // Initialize Logging before anything else happens
```

Comment:
Last I checked all our colors were colours. I'm pretty sure we use BE across the codebase